api/tests/Feature/CandidateAssessmentStatusTest.php

```diff
@@ -504,4 +505,124 @@ public function testOutOfOrderAssessmentsDoNotIncrementStep()
                 ],
             ]);
     }
+
+    public function testMultipleEssentialMustPassToIncrementStep()
+    {
+
+        $pool = Pool::factory()
+            ->published()
+            ->create([
+                'team_id' => $this->team->id,
+            ]);
+
+        $technicalSkills = Skill::where('category', SkillCategory::TECHNICAL->name)->limit(2)->get();
+        $poolSkillOne = PoolSkill::create([
+            'pool_id' => $pool->id,
+            'skill_id' => $technicalSkills[0]->id,
+            'type' => PoolSkillType::ESSENTIAL->name,
+        ]);
+
+        // Non-essential skills must also be assessed if a user claims them
+        $poolSkillTwo = PoolSkill::create([
+            'pool_id' => $pool->id,
+            'skill_id' => $technicalSkills[1]->id,
+            'type' => PoolSkillType::NONESSENTIAL->name,
```

Comment:
You called the test `testMultipleEssential` but the second skill is nonessential 😉

Reply:
Oof yeah, that test name should be more accurate after [adjust test names, order of creation](https://github.com/GCTC-NTGC/gc-digital-talent/pull/10925/commits/db93cca5019117064fe22794f3e8f8da2267b2a3)